src/menus/menu-divider.jsx

```diff
@@ -1,59 +1,36 @@
 import React from 'react';
-import StylePropable from '../mixins/style-propable';
-import ListDivider from '../lists/list-divider';
-import DefaultRawTheme from '../styles/raw-themes/light-raw-theme';
-import ThemeManager from '../styles/theme-manager';
+import Divider from '../divider';
+import Styles from '../utils/styles';
```

Comment:
I think that it should be in lower case. It's an utility object, not a class.
Maybe, we could call it `utilsStyles`.